man/resolved.conf.xml

```diff
@@ -269,6 +269,22 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes an IPv4 or IPv6 address to listen on. The address may optionally be prefixed by <literal>:</literal> and
+        a protocol name (<literal>udp</literal> or <literal>tcp</literal>). It may also be optionally suffixed by <literal>:</literal>
+        and a numeric port number. If the protocol is not specified, the service will listen on both IPv4 and IPv6. If the port is not
+        specified, then the service takes port as 53. This option may be used multiple times. Note that this is independent of the
+        primary DNS stub configured with <varname>DnsStubListener=</varname>, and only configures <emphasis>additional</emphasis>
+        sockets to listen on. Defaults to unset.</para>
+
+        <para>If the string in the format udp:[x]:y, it is read as protocol <literal>udp</literal> and IPv6 address x on a port y.</para>
```

Comment:
This needs markup:
`<literal>udp:[<replaceable>x</replaceable>]:<replaceable>y</replaceable>`, and then UDP, `<replaceable>x</replaceable>` and `<replaceable>y</replaceable>` further in the sentence.
(The protocol name is UDP, no quotes.)